README.md

```diff
@@ -183,11 +187,21 @@ function suggest (query, populateResults) {
 
 ### Other options
 
-#### `inputClasses` (default: `''`)
+#### `inputClasses` (default: `null`)
+
+Type: `string`
+
+Adds custom html classes to the `input` elements of the component:
+- the one the user types in
+- the one used to display the completion hint when `autoselect` is set to `true` (see `hintClasses` hereafter)
```

Comment:
Think we should link between `hintClasses` and `inputClasses`?

```suggestion
Adds custom html classes to the generated `input` element.

If [`autoselect`](#autoselect) is set to `true`, the option [`hintClasses`](#hintClasses) can be configured separately or it will default to the `inputClasses` value.
```